core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -787,6 +787,9 @@ sign_ext_32_64(int32 val)
 static inline void
 word_copy(uint32 *dest, uint32 *src, unsigned num)
 {
+    if (num == 0) {
```

Comment:
done.
i thought word_copy(.., NULL, 0) was ok because there were a few places calling the function that way. but i have no strong opinions either ways.